jwst/assign_wcs/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 from .assign_wcs_step import AssignWcsStep
 
-__version__ = '0.7.4'
+__version__ = '0.7.1.4'
```

Comment:
This doesn't look right, a typo?

Reply:
As mentioned in the description, I took cases where the existing version number - in this case 0.7.4 - was already ahead of 0.7.1 and tried to preserve it some fashion by moving the 3rd digit to a 4th digit, hence 0.7.4 becomes 0.7.1.4.

If folks don't want to use 4 digits where necessary (or think it'll actually cause problems), I can just set them all to 0.7.1 regardless of what their latest value had been.

Reply:
Sorry, I didn't see the initial description.
Perhpas it's time to start using semantic versioning and a scheme that's recognized in general. How about setting all versions to `0.8.0` and incresing the numbers in the future following rules for major.minor.patch. It has the benfit of communicating the significance of a release while tying the version to internal builds means nothing to most users. 
Perhaps software tools can take advantage of the semantic versioning as well.

Reply:
What do others think of Nadia's suggestion to just set all versions to 0.8.0 for B7.1, which could then go to 0.9.0 for B7.2, and eventually 1.0.0 for the first public release.

Reply:
i don't believe the software versions for anyone else in dms corresponds to the "build identifier" anyways.

Reply:
Seems reasonable to me.